### DISCUSSION TITLE
One more controller modification

### DISCUSSION THREAD

**Discussion by oppozith** [kachurovskiy/megaels#25] [Show and tell]
Jast an mod for original version
![20240228_171317](https://github.com/kachurovskiy/megaels/assets/5749538/6e152812-6297-4803-8cb7-dd9e11d89a92)

**Comment by kachurovskiy** [comment]
Very nice. Please attach relevant files that you modified such as Gerber amd 3D models. Thanks 🙏

**Comment by oppozith** [comment]
:)
Original case model + drill, buttons and hot glue ;)

**Comment by oppozith** [comment]
Just made case with sign marks on the plate. Will post print results asap
https://www.thingiverse.com/thing:6509165
![Screenshot 2024-02-29 121007](https://github.com/kachurovskiy/megaels/assets/5749538/6c02de45-2f3d-4a65-84f3-bf6c30fac012)

**Reply by oppozith** [reply]
![20240229_223830](https://github.com/kachurovskiy/megaels/assets/5749538/3acaeaf5-a093-445b-b0bf-1f090be58f8a)

**Reply by kachurovskiy** [reply]
Pretty cool! I had better luck printing on a regular printer though and then laminating.

**Comment by oppozith** [comment]
Another version![20240301_214040~2.jpg](https://github.com/kachurovskiy/megaels/assets/5749538/703d06b3-d165-4de5-b8ca-498cbf2e9489)
